src/piadm.sh

```diff
@@ -178,7 +178,7 @@ piname_present_get_bootfs() {
 # Defined as a variable in case we need to add parameters (like -s) to it.
 # WARNING:  Including -k for now.
 CURL=( curl -ks -f )
-VCURL=( curl -k -f --progress-bar )
+VCURL=( curl -k -f -S --progress-bar )
 
 vcurl() {
 	if [[ $VERBOSE -eq 1 ]]; then
```

Comment:
Below this there's a comment about verbose curls.  Please update it to reflect your new goodness here?